robotcontainer.py

```diff
@@ -41,19 +42,20 @@
 from commands.shootball import ShootBall
 from commands.defensestate import DefenseState
 
-from commands.indexer.holdball import HoldBall
 from commands.intake.autoballintake import AutoBallIntake
 from commands.intake.deployintake import DeployIntake
 from commands.intake.retractintake import RetractIntake
 from commands.shooter.aimshootertotarget import AimShooterToTarget
 from commands.shooter.aimshootermanual import AimShooterManually
-from commands.shooter.tarmacshot import TarmacShot
```

Comment:
Was this line supposed to be removed?

Reply:
Yes, it was an unused import.  Its possible that older version of pylint missed it.